setup.cfg

```diff
@@ -29,6 +30,9 @@ docs =
     sphinx<4.0
     sphinx-astropy
     jupyter-sphinx
```

Comment:
Maybe put the constraint on astropy here?

Reply:
Done